OQ_Toolkit/OQ_ARVROrigin/scripts/Locomotion_Stick.gd

```diff
@@ -18,7 +18,7 @@ export(vr.AXIS) var move_left_right = vr.AXIS.LEFT_JOYSTICK_X;
 export(vr.AXIS) var move_forward_back = vr.AXIS.LEFT_JOYSTICK_Y;
 
 enum MovementOrientation { HEAD, HAND_LEFT, HAND_RIGHT }
-export(MovementOrientation) var movmenet_orientation := MovementOrientation.HEAD
+export(MovementOrientation) var movemnet_orientation := MovementOrientation.HEAD
```

Comment:
Still swapped letters :-); it should read movement_orientation

could you also squash your commits after you fixed this (so that only a single commit is in the pull request; this is not really relevant here at the moment since this is a small project with only few contributions so far; but it is common practice for larger repos like godot to keep the history cleaner: https://docs.godotengine.org/en/3.2/community/contributing/pr_workflow.html)

Reply:
OK it should actually be fixed now lol, also squashed